source/framework/core/inc/TRestAnalysisTree.h

```diff
@@ -266,6 +281,93 @@ class TRestAnalysisTree : public TTree {
 
     Int_t WriteAsTTree(const char* name = 0, Int_t option = 0, Int_t bufsize = 0);
 
+    Bool_t AddChainFile(std::string file);
+
+    TTree* GetTree() const;
+
+    TChain* GetChain() { return fChain; }
+
+    Long64_t LoadTree(Long64_t entry);
+
+    Long64_t GetEntries() const;
+
+    Long64_t GetEntries(const char* sel);
+
+    void Browse(TBrowser* b) { fChain ? fChain->Browse(b) : TTree::Browse(b); }
+    Long64_t Draw(const char* varexp, const TCut& selection, Option_t* option = "",
+                  Long64_t nentries = kMaxEntries, Long64_t firstentry = 0) {
+        return fChain ? fChain->Draw(varexp, selection, option, nentries, firstentry)
+                      : TTree::Draw(varexp, selection, option, nentries, firstentry);
+    }
+    Long64_t Draw(const char* varexp, const char* selection, Option_t* option = "",
```

Comment:
Yes. It is used to read splitted data files. In case multiple files are created to store analysis tree data, we make TRestAnalysisTree running internally with TChain. All the functions called to TRestAnalysisTree is calling back the embed TChain object.